paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/FormUI.kt

```diff
@@ -59,43 +38,13 @@ internal fun FormInternal(
     elementsFlow: Flow<List<FormElement>?>,
     lastTextFieldIdentifierFlow: Flow<IdentifierSpec?>
 ) {
-    val hiddenIdentifiers by hiddenIdentifiersFlow.collectAsState(emptyList())
-    val enabled by enabledFlow.collectAsState(true)
-    val elements by elementsFlow.collectAsState(null)
-    val lastTextFieldIdentifier by lastTextFieldIdentifierFlow.collectAsState(null)
-
-    Column(
-        modifier = Modifier.fillMaxWidth(1f)
-
+    FormUI(
+        hiddenIdentifiersFlow,
+        enabledFlow,
+        elementsFlow,
+        lastTextFieldIdentifierFlow
     ) {
-        elements?.let {
-            it.forEachIndexed { _, element ->
-                if (!hiddenIdentifiers.contains(element.identifier)) {
-                    when (element) {
-                        is SectionElement -> SectionElementUI(
-                            enabled,
-                            element,
-                            hiddenIdentifiers,
-                            lastTextFieldIdentifier
-                        )
-                        is StaticTextElement -> StaticElementUI(element)
-                        is SaveForFutureUseElement -> SaveForFutureUseElementUI(enabled, element)
-                        is AfterpayClearpayHeaderElement -> AfterpayClearpayElementUI(
-                            enabled,
-                            element
-                        )
-                        is AuBecsDebitMandateTextElement -> AuBecsDebitMandateElementUI(element)
-                        is AffirmHeaderElement -> AffirmElementUI()
-                        is MandateTextElement -> MandateTextUI(element)
-                        is CardDetailsSectionElement -> CardDetailsSectionElementUI(
-                            enabled, element.controller, hiddenIdentifiers
-                        )
-                        is BsbElement -> BsbElementUI(enabled, element, lastTextFieldIdentifier)
-                        is EmptyFormElement -> {}
-                    }
-                }
-            }
-        } ?: Row(
+        Row(
```

Comment:
can we make this clearer that this is a loadingComposable? It looks like this is what is supposed to be rendered.

Reply:
Done